tests/clip/test_processor_clip.py

```diff
@@ -86,19 +89,28 @@ def prepare_image_inputs(self):
         return image_inputs
 
     def test_save_load_pretrained_default(self):
-        tokenizer = self.get_tokenizer()
+        tokenizer_s = self.get_tokenizer()
+        tokenizer_r = self.get_rust_tokenizer()
```

Comment:
(nit) Maybe call this `tokenizer_slow` and `tokenizer_fast`

Reply:
actually I use the name refering to `test_tokenization_clip.py`
like https://github.com/huggingface/transformers/blob/a7df656f0343046736a71d4b2218769db92977eb/tests/clip/test_tokenization_clip.py#L82-L83

but I agree tokenizer_slow and tokenizer_fast is better.

Reply:
I modified it!